apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/TableSchemaBuilder.cs

```diff
@@ -40,10 +41,30 @@ public sealed class TableSchemaBuilder : IEnumerable
         public void Add(TableFieldSchema field)
         {
             GaxPreconditions.CheckNotNull(field, nameof(field));
-
+            _tableFields[field.Name] = field;
             _fields.Add(field);
         }
 
+        /// <summary>
+        /// Modifies the given fields as per the specified action.
+        /// </summary>
+        /// <param name="fieldName">The name of the field to be modified. Must not be null.</param>
+        /// <param name="modifyField">An Action to be performed on the given field.</param>
```

Comment:
Nit: Action => action (as if we were really trying to specify the type name, it would need the generic type).